relayer/application-standard.ts

```diff
@@ -58,10 +59,16 @@ export class StandardRelayerApp<
     const logger = opts.logger;
     delete opts.logger;
     // now we can merge
-    opts = mergeDeep({},defaultOpts, opts);
+    opts = mergeDeep({}, defaultOpts, opts);
```

Comment:
this will have conflicts with [Joe PR](https://github.com/gabzim/wormhole-relayer/pull/13/files#diff-01c3f5d8b2e9eebd1441f6da69a257153c8fc44933f793e6b2689378263346b1R63)